civihr_employee_portal/js/reports.js

```diff
@@ -579,7 +575,6 @@
       return false;
     }
     var reportName = this.reportName;
-    var configId = this.getReportConfigurationId();
```

Comment:
Ware you sure you wont need it?

Reply:
```
var configId = this.getReportConfigurationId();
if (!configId) {
  swal('No configuration selected', 'Please choose configuration to delete.', 'error');
  return false;
}
var reportName = this.reportName;
var configId = this.getReportConfigurationId();
```
Line 1 and 7 is exactly same, so we were initialising the same var with same value twice. And between the 2 lines, no other line is changing the value of `getReportConfigurationId()`, so it does not make sense to keep both lines.